### DISCUSSION TITLE
Binding Errors

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [MahApps/MahApps.Metro#4232] [Q&A] [answered]
I'm getting a lot of binding errors, i'm not overriding any style of the datagrid
![error](https://user-images.githubusercontent.com/6304701/146836953-663cb273-27e7-4fc3-a18d-fa3967e84ef4.jpg)

Could this [line](https://github.com/MahApps/MahApps.Metro/blob/a0f8dc3389a0d1171813a2057c3bf8b3478c9cbd/src/MahApps.Metro/Styles/Controls.DataGrid.xaml#L246) hold the problem?

**Comment by punker76** [comment, marked as answer]
The relative binding to the DataGridRow from a DataGridCell is the only way to do this, because the property is only available internal.

```csharp
        internal DataGridRow RowOwner
        {
            get { return _owner; }
        }
```

As I can see, these binding errors (warnings) only occurs when the user set the visibility of the cell to collapse (hide). And the binding error level is set to warning.

So maybe we can live with that...
